src/sentry/models/relay.py

```diff
@@ -24,7 +25,7 @@ class Meta:
 
 @region_silo_only_model
 class Relay(Model):
-    __include_in_export__ = True
+    __relocation_scope__ = RelocationScope.Global
```

Comment:
Relay configurations are generally Organization bound as the organization has an option containing a list of trusted relay instances.

Reply:
I was a bit confused about this, since I couldn't find any fields on the model that reference orgs?

Reply:
An organization's relays are stored as an organziation option so there isn't any schema :cry: